FreeRTOS/FreeRTOS-Kernel
# RISC-V: No #error on RV64 regarding byte alignment

RISC-V: No #error on RV64 regarding byte alignment

Description
-----------
Remove `#error` that is triggered on 64-bit RISC-V compiles. No functional change.

I tried to find a definitive source for alignment requirements, but things are a bit convoluted. See https://github.com/riscv/riscv-elf-psabi-doc/issues/21.

Test Steps
-----------
I encountered no issues on a Kendryte K210 with 16 byte alignment.


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
